src/Files.App/Data/Items/DriveItem.cs

```diff
@@ -152,6 +177,28 @@ public bool ShowStorageSense
 
 		public string Name => Root.DisplayName;
 
+		public object? Children => null;
+
+		private object toolTip = "";
+		public object ToolTip
+		{
+			get => toolTip;
+			set
+			{
+				SetProperty(ref toolTip, value);
+			}
+		}
+
+		public bool IsExpanded { get => false; set { } }
```

Comment:
Is it worth adding a comment explaining what IsExpanded does?

Reply:
Very good point, I went ahead and just wrote some more documentation in general (including for this property).

Reply:
Thank you